app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,8 @@
       OX.url_prefix = "<%= request.script_name %>";
     </script>
 
+    <%= render partial: "layouts/google_tag_manager" %>
```

Comment:
Did you mean to add this the new flow? https://github.com/openstax/accounts/blob/master/app/views/layouts/newflow_layout.html.erb

Only adding it to app/views/layouts/application.html.erb will make it only available in the legacy flow and users' profile.

Reply:
Good point. Fixed